tests/common.py

```diff
@@ -487,7 +487,7 @@ def __init__(self, domain=None, dependencies=None, setup=None,
     def mock_manifest(self):
         """Generate a mock manifest to represent this module."""
         return {
-            **loader.manifest_from_legacy_module(self),
+            **loader.manifest_from_legacy_module(self.DOMAIN, self),
```

Comment:
why legacy_module?

Reply:
it's generating a manifest from a Python module. The new approach for integrations is that manifests are their own file.